test-infra/aws-vpn/README.md

```diff
@@ -0,0 +1,80 @@
+# aws-vpn infrastructure example
+
+This example will provision a VPC on AWS, together with an EKS cluster that's private to the VPC and a ClientVPN endpoint to access it.
+This is basically all that is needed for a private EKS cluster inside a VPC, and can be used to test how telepresence interacts with different VPN scenarios.
+
+## How to use it
+
+### 0. Prerequisites
+
+You will need a route53 zone in your AWS account.
```

Comment:
Ignorant question: do we have a route53 zone in the datawire AWS account? Or were you using your personal account?

Reply:
Indeed we do -- the way this terraform configuration uses route53 actually comes from our own [best practices](https://www.notion.so/datawire/Create-Domains-With-Route53-f63874a74b7a4f9ca2816dd18b5105a5#55ba3374352d4f8084c54d9f3bca8a85) around creating new domains, but I wasn't sure if it was a good idea to link to our internal notion in a public README

Reply:
Yup totally makes sense not to include that. Ty!